receiver/prometheusremotewritereceiver/receiver_test.go

```diff
@@ -323,8 +321,8 @@ func TestTranslateV2(t *testing.T) {
 				parseJobAndInstance(rm.Resource().Attributes(), "service-z/xyz", "inst-42")
 				sm := rm.ScopeMetrics().AppendEmpty()
 				// Expect fallback to default BuildInfo.
-				sm.Scope().SetName("")
-				sm.Scope().SetVersion("")
+				sm.Scope().SetName(defaultBuildName)
+				sm.Scope().SetVersion(defaultBuildVersion)
```

Comment:
Using the empty string instead of defaultBuildName/Version does the test fail?

Reply:
Hey @Vandit1604 I think that is only thing that is missing before we merge it 🙃

Reply:
And some conflicts related to [this PR](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/37895/files#diff-e26ac1e595473c2a7f1cf349aee9b1cceced9c8f9c4615b08be8b9e551edfcd9) that was merged recently